### DISCUSSION TITLE
Remove display name

### DISCUSSION THREAD

**Discussion by thatjar** [community/community#22530] [New to GitHub]
<p>I’m unable to remove my name on GitHub Support Community. The username is fine but I am trying to remove the text in “your full name (optional)”. Please let me know how I can remove it.</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>:wave: Hey - welcome back.</p>
<p>I’ve just gone ahead and removed that for you.</p>
<p>Out of curiosity - do you know where in your GitHub profile that was picked up?</p>

**Comment by thatjar** [comment]
<p>Thanks!</p>
<aside class="quote group-github-employees" data-username="canuckjacq" data-post="2" data-topic="147237">
<div class="title">
<div class="quote-controls"></div>
 canuckjacq:</div>
<blockquote>
<p>Out of curiosity - do you know where in your GitHub profile that was picked up?</p>
</blockquote>
</aside>
<p>That was actually my first username on GitHub. I have changed my username a few times.</p>

**Comment by canuckjacq** [comment]
<p>that makes sense! Thanks for indulging my curiosity on that - you saved me some testing.</p>

**Comment by ferg1e** [comment]
<p>I would like to remove my real name as well. I also see “your full name (optional)” but don’t see how to remove it. Thank you.</p>

**Comment by canuckjacq** [comment]
<p>I’m pretty sure that’s picked up from your GitHub profile, so you’d need to log out of the forums, remove/change it from your GitHub profile, and then log back in. That usually re-syncs the user information.</p>

**Comment by BeansBeefBroccoli** [comment]
<p>My name is no longer associated with my GitHub profile, but logging out and in didn’t work.</p>

**Comment by hone1er** [comment]
<p>I am trying to remove my name from my main github profile</p>
<p>I still see hone1er(full name) when searching for my profile when i would like it to be just hone1er</p>
<p>I have checked every tab in the settings and my full name is nowhere to be found yet it is still attached to my profile somehow</p>

**Comment by hone1er** [comment]
<p>Hmm it seems to have been resolved for me but when searching on a new device i still see my old name and photo</p>
<p>I’ve logged out and back in but it still shows up on search</p>
<p>EDIT: Not sure if the issue lies with google but when i use Brave search it is fixed, no full name, but in google it still shows up as the attached photo. It has been about 2 months since i changed the name and profile picture from what is being shown by google(and bing)</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/b/7/b76371abc3e738dd80d14f77db34ab0c20400e60.jpeg" data-download-href="/uploads/short-url/qakqXTR1B9xCfzHTSRQymagTvdS.jpeg?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/24376928/181071667-13496ec8-e394-49aa-90bc-ed3bea8d1608.jpeg" alt="image" data-base62-sha1="qakqXTR1B9xCfzHTSRQymagTvdS" width="230" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/b/7/b76371abc3e738dd80d14f77db34ab0c20400e60_2_230x500.jpeg, https://aws1.discourse-cdn.com/github/optimized/3X/b/7/b76371abc3e738dd80d14f77db34ab0c20400e60_2_345x750.jpeg 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/b/7/b76371abc3e738dd80d14f77db34ab0c20400e60_2_460x1000.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/b/7/b76371abc3e738dd80d14f77db34ab0c20400e60_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">945×2048 110 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by SSBladedge** [comment]
Hi I would also like to remove the name in parenthesis from my profile, please  
I tried changing it in settings and using git config, but it moved to parenthesis and won't go away
